### PR TITLE
Add the validator contact form in javaScript

### PR DESCRIPTION
**Pull request by Nejjari001**
implement a client-side validation of the data in the contact form before submission

### CODE REVIEW

File: validate.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    fname.value.length > 3
+    && fname.value.length < 30
+    && message.value.length > 10
+    && message.value < 400
+    && emailRegex.test(email.value)
+  ) {
+    form.submit();
+  }
+  return false;
+});
```
Comment (zieeco) [inline validate.js]: - Hey, great job, however, the form is still being submitted when the email is in Uppercase. please, kindly fix this to ensure the form doesn't get submitted if the email is in UpperCase. e.g `YOU@GMAIL.COM`

Kindly check the video to see the form in action :point_down: 

https://user-images.githubusercontent.com/59397159/190417051-63f4f829-e480-4876-addb-3c56f63476a9.mp4

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far! 👏 
You've done it well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: validate.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  } else if (fname.value === '' || email.value === '' || message.value === '') {
+    overralError.textContent = 'Please fill in entire form before submitting.';
+    document.getElementById('field-overal').setAttribute('class', 'error');
+    event.preventDefault();
+  } else {
+    overralError.textContent = '';
+    overralError.style.display = 'none';
+    event.preventDefault();
+  }
+});
```
Comment (julie-ify) [inline validate.js]: - [ ] I am still able to successfully submit the form with upper cased email. Kindly resolve that.

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @Nejjari001,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Nejjari001 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Correct use of GitHub flow. ✔️ 
 - Professional PR message. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**